tony-history-server/app/cache/CacheWrapper.java

```diff
@@ -2,16 +2,18 @@
 
 import com.google.common.cache.Cache;
 import com.google.common.cache.CacheBuilder;
+import java.util.List;
 import javax.inject.Singleton;
 import models.JobConfig;
 import models.JobMetadata;
+import org.apache.hadoop.fs.Path;
 import play.Logger;
 
 @Singleton
 public class CacheWrapper {
   private static final Logger.ALogger LOG = Logger.of(CacheWrapper.class);
   private static Cache<String, JobMetadata> metadataCache;
-  private static Cache<String, JobConfig> configCache;
+  private static Cache<String, List<JobConfig>> configCache;
```

Comment:
Please add documentation explaining what the keys and values in each cache represent.